frontend/src/components/App/Settings/SettingsCluster.tsx

```diff
@@ -168,6 +168,13 @@ export default function SettingsCluster() {
             : t('settings|Cluster Settings')
         }
         backLink
+        headerProps={{
+          actions: [
+            <Link href="/settings" align="right" color="inherit">
```

Comment:
We shouldn't hardcode the URLs since we want to have only one source of truth for them (this way we can change in one place and have the changes be reflected throughout the app).
So the URL should be gotten by using `createRouteURL('settings')`.
I also realize now that if you use our own `Link` component from `common`, you can just do `<Link routeName="settings">...`

Reply:
Will use createRoute!